libzfs.pyx

```diff
@@ -2644,7 +2649,7 @@ cdef class ZPoolScrub(object):
             if not self.bytes_to_scan:
                 return 0
 
-            return (<float>self.bytes_issued / <float>self.bytes_to_scan) * 100
+            return (<float>self.bytes_issued / (<float>self.bytes_to_scan - <float>self.bytes_skipped)) * 100
```

Comment:
In zpool I also check for bytes_to_scan > bytes_skipped to avoid division by zero, but it should probably be extremely rare, if ever happen.